src/ruby/ext/symengine/ruby_symbol.h

```diff
@@ -0,0 +1,9 @@
+#include "ruby.h"
+#include "ruby_basic.h"
+#include "cwrapper.h"
+
+void csymbol_free(void *ptr);
```

Comment:
No, it isn't. But it might be required in future, if I use `Data_Make_Struct` to make a new object of `Symbol` in ruby wrapper code. I can remove it for the time being.